extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -25,22 +25,61 @@ define('ext.wikia.adEngine.lookup.services', [
 			rubiconFastlane,
 			prebid,
 			rubiconVulcan
-		];
+		],
+		bidIndex = {
+			'rubicon_fastlane': {
+				'pos': 0,
+				'char': 'R'
+			},
+			'ox_bidder': {
+				'pos': 1,
+				'char': 'O'
+			},
+			'amazon': {
+				'pos': 2,
+				'char': 'A'
+			},
+			'rubicon_vulcan': {
+				'pos': 3,
+				'char': 'V'
+			},
+			'prebid': {
+				'pos': 4,
+				'char': 'P'
+			}
+		},
+		bidMarker = ['x','x','x','x','x'];
```

Comment:
I wonder if it's worth to have this initial bidMarker auto generated' base on the number of bidders, but maybe I'm trying to overengineer.